sympy/matrices/dense.py

```diff
@@ -1450,7 +1450,7 @@ def wronskian(functions, var, method='bareiss'):
     if n == 0:
         return 1
     W = Matrix(n, n, lambda i, j: functions[i].diff(var, j))
-    return W.det(method)
+    return W.det(method='bareiss')
```

Comment:
I think this should be `return W.det(method=method)`. The wronskian accepts the keyword argument method. It should pass it along. If you hardcode `bareiss` here, that's ignoring the argument given to wronskian.  

The variable `method` will have 'bareiss' in it if wronskian doesn't get that keyword argument, since it's the default value